analyzer/pkg/rpm/rpm.go

```diff
@@ -95,6 +95,7 @@ func (a rpmPkgAnalyzer) parsePkgInfo(packageBytes []byte) (pkgs []types.Package,
 			SrcVersion:      srcVer,
 			SrcRelease:      srcRel,
 			Modularitylabel: pkg.Modularitylabel,
+			License:         strings.Replace(strings.Replace(pkg.License, " and ", ",", -1), " or ", ",", -1),
```

Comment:
I don't think "and" and "or" are the same. Is there any specification on the license field in RPM DB? If no, we should keep this field as is.